models/public/mozilla-deepspeech-0.6.1/accuracy-check-fast.yml

```diff
@@ -0,0 +1,59 @@
+#
+# This config file requires ctcdecode_numpy Python module for fast_ctc_beam_search_decoder_with_lm.
+# Please see open_model_zoo/demos/python_demos/speech_recognition_demo/README.md for installation instructions.
+#
+models:
+  - name: mozilla-deepspeech-0.6.1-with-lm
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        adapter:
+          type: fast_ctc_beam_search_decoder_with_lm
+          probability_out: logits
+          logarithmic_prob: False
+          beam_size: 500
+          lm_file: ../deepspeech-0.6.1-models/lm.binary
```

Comment:
```suggestion
          lm_file: lm.binary
```

Reply:
The reason for relative path here is that lm.binary is not in the same directory as the converted model (.xml and .bin files). Running `<path>/downloader.py --name mozilla-deepspeech-0.6.1` and `<path>/converter.py --name=mozilla-deepspeech-0.6.1` creates directory structure where these two files are in brother (sibling) directories: one in FP32/ subdirectory, and another in deepspeech-0.6.1-models/ subdirectory. With relative path it is enough to use `--models` option, and AC will find LM with the default directory structure. The `--model_attributes` option is close to being undocumented, and it'll be hard for a new user to understand what it means from its name. Even without it I had a hard time running AC for the first time.

Did I convince you? In case you insist, I'll change it.

Reply:
Model can be downloaded and converted to the different directories, in this case, your path will not work. Also I am not sure that it can successfully processed in our inrenal infra.
Ok, I check docs and if info about --model_attributes really is not provided, I'll update it

Reply:
OK, I'll remove it.

Description of --model_attributes doesn't explain what is meant by "model attributes". I think it can be improved by: 1) moving the line with --model_attributes up (in accuracy_checker/README.md) to the list with --models, --source and --annotations, and 2) extending it with some examples like "(for example, vocabulary files, \<some other\> files, etc.)".

Reply:
I already removed the relative path in the later commit. You can switch from "Changes from <number> commits" to "Changes from all commits" in github's "Files changed" view top-left, and search for filename to check.